src/libraries/Directory.Build.props

```diff
@@ -3,21 +3,20 @@
     <SkipImportArcadeSdkFromRoot>true</SkipImportArcadeSdkFromRoot>
   </PropertyGroup>
   <Import Project="..\..\Directory.Build.props" />
+  <Import Project="NetCoreAppLibrary.props" />
 
   <PropertyGroup Condition="$(MSBuildProjectName.StartsWith('Microsoft.Extensions.'))">
     <IsAspNetCoreApp>true</IsAspNetCoreApp>
   </PropertyGroup>
 
   <PropertyGroup>
     <BeforeTargetFrameworkInferenceTargets>$(RepositoryEngineeringDir)BeforeTargetFrameworkInference.targets</BeforeTargetFrameworkInferenceTargets>
-    <ToolSetCommonDirectory>$(RepoRoot)artifacts\toolset\Common\</ToolSetCommonDirectory>
+    <CustomAfterMicrosoftCommonCrossTargetingTargets>$(RepositoryEngineeringDir)CapturedBuildOutputCrossTargetingTargets.targets</CustomAfterMicrosoftCommonCrossTargetingTargets>
```

Comment:
Would it make sense to do this in our TargetFramework SDK?

Reply:
I don't think this should go into our TargetFramework.Sdk as it isn't related to multi-targeting by runtime.

Reply:
Fair, if we think MSBuild will fix it then I'm not too concerned.